infra/scripts/dbs/rds/mysql/create.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+GIT_TLD=`git rev-parse --show-toplevel`
+source ${GIT_TLD}/scripts/dbs/rds/common.sh
```

Comment:
please update the path to source ${GIT_TLD}/infra/scripts/dbs/rds/common.sh
because scripts are under infra folder